Orbitersdk/samples/ProjectApollo/src_sys/yaAGC/agc_engine.c

```diff
@@ -1749,6 +1749,8 @@ agc_engine (agc_t * State)
 		  // Same with Standby
 		  if (0 == (State->InputChannel[032] & 020000))
 			  State->SbyPressed = 1;
+		  else
+			  State->SbyPressed = 0;
```

Comment:
I recommend moving this check outside of the `while (ScalerCounter >= SCALER_OVERFLOW)` loop. Placing it here makes the check for PRO release only occasionally happen based on the current scaler time; in reality there's no clocks gating the reset so it happens immediately upon release. So you might get some slightly weird timing effects. Putting the check to do this just before the while loop should be fine.